cmd/otelcontribcol/go.mod

```diff
@@ -1069,3 +1069,6 @@ replace github.com/bradfitz/gomemcache => github.com/grafana/gomemcache v0.0.0-2
 
 // fork containing a line-buffered logger which should improve logging performance for loki
 replace github.com/go-kit/log => github.com/dannykopping/go-kit-log v0.2.2-0.20221002180827-5591c1641b6b
+
+// see https://github.com/mattn/go-ieproxy/issues/45
```

Comment:
Based on the comments should revert to v0.0.7

Reply:
I tried them all, its failing starting from 0.0.2